pkg/client/model_serverless_cluster_update_specification.go

```diff
@@ -40,16 +40,16 @@ func NewServerlessClusterUpdateSpecification() *ServerlessClusterUpdateSpecifica
 
 // GetSpendLimit returns the SpendLimit field value if set, zero value otherwise.
 func (o *ServerlessClusterUpdateSpecification) GetSpendLimit() int32 {
-	if o == nil || o.SpendLimit == nil {
+	if o == nil {
 		var ret int32
 		return ret
 	}
-	return *o.SpendLimit
+	return o.SpendLimit.GetValue()
 }
 
 // SetSpendLimit gets a reference to the given int32 and assigns it to the SpendLimit field.
-func (o *ServerlessClusterUpdateSpecification) SetSpendLimit(v int32) {
-	o.SpendLimit = &v
+func (o *ServerlessClusterUpdateSpecification) SetSpendLimit(v *int32, marshalNull bool) {
```

Comment:
Ideally, the API would "make simple things simple and complex things possible". For example, could we have an API like this instead:
```
// Set spend limit (the default Set method does not take a pointer).
spec.SetSpendLimit(1)

// No spend limit set (this would set marshalNull=true).
spec.ClearSpendLimit()
```